src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -894,7 +894,7 @@ trait BronzeTransforms extends SparkSessionWrapper {
   private[overwatch] def getAllEventLogPrefix(inputDataframe: DataFrame, apiEnv: ApiEnv): DataFrame = {
     try{
       val mountMap = getMountPointMapping(apiEnv) //Getting the mount info from api and cleaning the data
-        .filter(col("mount_point") =!= "/")
+        .filter(col("mount_point") =!= "/").filter(trim(col("mount_point")) =!= "") //Added for #1021
```

Comment:
@sriram251-code  Instead of 2 filter condition can we put both as and condition. and also if you are adding trim in one filter condition please add in both.